cms/admin/forms.py

```diff
@@ -389,10 +385,26 @@ def save(self, *args, **kwargs):
             new_page = self.from_source(source, parent=parent)
         else:
             new_page = Page(site=self._site, parent=parent)
-            new_page.add_to_tree(position='last-child')
+            new_page.add_to_tree(position="last-child")
             new_page.save()
 
-        translation = self.create_translation(new_page)
+        # If you have djangocms-versioning installed, this will get the latest version of the page content, and ignore drafts.
+        # Get the original page content template if available
+        og_page_content_template = None
+        if new_page:
+            try:
+                main_language = new_page.get_languages()[0]
+                og_page_content = (
+                    PageContent.objects.filter(page_id=new_page.pk, language=main_language).only("template").first()
```

Comment:
```suggestion
                    new_page.get_admin_content(language=main_language)
```

There's an implicit design decision here: 
* You'll get the page's latest (public) content object. The preferred way of doing this would be `new_page.get_content_object(language=main_language)`
* Alternatively, since we're operating in the admin, you could get the latest admin object (see suggestion above). This will also return an admin object if there (with versioning) there is no public page content

Reply:
Also, what's `og` supposed to stand for?

Reply:
Nice suggestion! I decided to go with your alternative and use `get_admin_content`. However, there's an important behavior decision to consider, and I’d love your input on it:

By using this function, the translation will always fetch its template from the draft version of the main language page content (if one exists), rather than the published version. If no draft is available, it will fall back to the public version.

IMO that logic makes sense, I just need to adjust the docs to reflect the behavior. What do you think about this?

> Also, what's og supposed to stand for?
og was supposed to stand for original, but I see it's not really self-explanatory even when using the word "original", so I changed it to "main_language" instead :) Thanks for pointing it out!

Reply:
Using the latest draft is the default behaviour throughout the admin. I second your decision.